spec/tests/result.fmf

```diff
@@ -23,8 +23,9 @@ description: |
         test needs to create it's own ``results.yaml`` file in the
         ``${TMT_TEST_DATA}`` directory. The ``results.yaml`` contains list of
         dictionaries. Each dictionary must contain at least ``name`` and
```

Comment:
Could you mention that '/' is name used to provide result for itself ?

Reply:
@happz seems one comment prevents the merge in this one

Reply:
Added in eb40e829.